dockerfiles/mongodb-agent/107.0.1.8507-1/ubi/Dockerfile

```diff
@@ -1,7 +1,7 @@
 ARG imagebase
 FROM ${imagebase} as base
 
-FROM registry.access.redhat.com/ubi8/ubi-minimal
```

Comment:
2 sanity checks:
- have we checked that we didn't overwrite existing ubi8 images with ubi9?
- why is there not `agent_1.28.0` dockerfile images here?

Reply:
* First one, yes, we don't override existing ubi8 images with ubi9
* It looks like we have a race condition with the dockerfiles: we don't have the agents for the new operator as we build the dockerfiles.